docs/loading/Loading_data_template.md

```diff
@@ -0,0 +1,348 @@
+     1|3745169|   2891509|pv          |2017-11-26 19:44:31|
+     1|1531036|   2920476|pv          |2017-11-26 22:02:12|
+     1|2266567|   4145813|pv          |2017-11-27 00:11:11|
+     1|2951368|   1080785|pv          |2017-11-27 02:47:08|
+```
+
+#### Creating a table with schema inference
+
+> Note
+>
+> The syntax of `CREATE TABLE` when using schema inference does not allow setting the number of replicas, so set it before creating the table. The example below is for a system with a single replica:
+>
+> `ADMIN SET FRONTEND CONFIG ('default_replication_num' ="1");`
+
+```sql
+CREATE DATABASE IF NOT EXISTS project;
+USE project;
+
+CREATE TABLE `user_behavior_inferred` AS
+SELECT * FROM FILES(
+    "path" = "s3://starrocks-datasets/user_behavior_sample_data.parquet",
+    "format" = "parquet",
+    "aws.s3.region" = "us-west-1",
+    "aws.s3.access_key" = "AAAAAAAAAAAAAAAAAAAA",
+    "aws.s3.secret_key" = "BBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB"
+);
+```
+
+```SQL
+DESCRIBE user_behavior_inferred;
+```
+
+```plaintext
+Field       |Type            |Null|Key  |Default|Extra|
+------------+----------------+----+-----+-------+-----+
+UserID      |bigint          |YES |true |       |     |
+ItemID      |bigint          |YES |true |       |     |
+CategoryID  |bigint          |YES |true |       |     |
+BehaviorType|varchar(1048576)|YES |false|       |     |
+Timestamp   |varchar(1048576)|YES |false|       |     |
+```
+
+> Note
+>
+> Compare the inferred schema with the schema created by hand:
+>
+> - data types
+> - nullable
+> - key fields
+
+```SQL
+SELECT * from user_behavior_inferred LIMIT 10;
+```
+
+```plaintext
+UserID|ItemID|CategoryID|BehaviorType|Timestamp          |
+------+------+----------+------------+-------------------+
+171146| 68873|   3002561|pv          |2017-11-30 07:11:14|
+171146|146539|   4672807|pv          |2017-11-27 09:51:41|
+171146|146539|   4672807|pv          |2017-11-27 14:08:33|
+171146|214198|   1320293|pv          |2017-11-25 22:38:27|
+171146|260659|   4756105|pv          |2017-11-30 05:11:25|
+171146|267617|   4565874|pv          |2017-11-27 14:01:25|
+171146|329115|   2858794|pv          |2017-12-01 02:10:51|
+171146|458604|   1349561|pv          |2017-11-25 22:49:39|
+171146|458604|   1349561|pv          |2017-11-27 14:03:44|
+171146|478802|    541347|pv          |2017-12-02 04:52:39|
+```
+
+#### Loading into an existing table
+
+By querying the data directly from S3 it is possible to decide the:
+
+- column data types
+- column names
+- columns to use in the key
+
+```SQL
+CREATE TABLE `user_behavior_declared` (
+    `UserID` int(11),
+    `ItemID` int(11),
+    `CategoryID` int(11),
+    `BehaviorType` varchar(65533),
+    `Timestamp` datetime
+) ENGINE=OLAP 
+DUPLICATE KEY(`UserID`)
+DISTRIBUTED BY HASH(`UserID`)
+PROPERTIES (
+    "replication_num" = "1"
+);
+```
+
+After creating the table, you can load it with `INSERT INTO` … `SELECT FROM FILES()`:
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/065731f6-d9ae-41f7-8a68-bbd5f3ec5470)
"it" refers to "the table" in "After creating the table"? But that doesn't make sense and it should be "the data"?